pytest_reportportal/plugin.py

```diff
@@ -164,8 +164,9 @@ def pytest_configure(config):
         config._reportportal_configured = False
         return
 
-    project = config.getini('rp_project') if config.getini('rp_project') \
-        else config.option.rp_project
+    project = config.option.rp_project
+    if not project:
+        project = config.getini('rp_project')
```

Comment:
```suggestion
        config.option.rp_project = project = config.getini('rp_project')
```
Once you do that, u'll be able to use session.config.option.rp_project in the pytest_sessionstart hook. pytest_configure hooks start before the pytest_sessionstart one.

Reply:
that's what I'm seeing:
config.option.rp_project = project = config.getini('rp_project')

That's what I see. I don't understand what do you need me to do as it's not valid python code

Reply:
```
Python 3.6.5 (default, Apr 10 2018, 17:08:37) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-16)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = b = 300
>>> a
300
>>> b
300
>>> 
```